axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/PersistentStreamConnection.java

```diff
@@ -59,6 +59,8 @@
  */
 public class PersistentStreamConnection {
 
+    public static final int MAX_RETRY_INTERVAL_SECONDS = 60;
+    public static final int MIN_RETRY_INTERVAL_SECONDS = 1;
```

Comment:
Are these constants intended to be `public`? If yes, they need `JavaDoc`. If no, we can make them `private`.